tox.ini

```diff
@@ -1,36 +1,37 @@
 [tox]
-envlist = py27-lint, py27-lint-imports, py27-lint-imports-include-list, py27-unit, qunit, mako-count, web-controller-line-count, py33-lint, py34-lint, py35-lint, validate-test-tools, py27-lint-docstring-include-list, py27-lint-docstring
+# envlist is the list of envs that are tested when `tox` is run without any option
+envlist = first_startup, py27-lint, py27-lint-docstring-include-list, py27-lint-imports-include-list, py27-unit, py34-lint, qunit, validate-test-tools
 skipsdist = True
 
-[testenv:py27-lint]
-commands = bash .ci/flake8_wrapper.sh
-whitelist_externals = bash
-deps = -rlib/galaxy/dependencies/pipfiles/flake8/pinned-hashed-requirements.txt
+[testenv:check-python-dependencies]
+commands = make list-dependency-updates # someday change exit code on this.
+whitelist_externals = make
+skip_install = True
 
-[testenv:py33-lint]
-commands = bash .ci/flake8_wrapper.sh
+[testenv:first_startup]
+commands = bash .ci/first_startup.sh
 whitelist_externals = bash
-deps = -rlib/galaxy/dependencies/pipfiles/flake8/pinned-hashed-requirements.txt
 
+[testenv:mako-count]
+commands = bash .ci/check_mako.sh
+whitelist_externals = bash
 
-[testenv:py34-lint]
+[testenv:py27-lint]
 commands = bash .ci/flake8_wrapper.sh
 whitelist_externals = bash
 deps = -rlib/galaxy/dependencies/pipfiles/flake8/pinned-hashed-requirements.txt
 
-[testenv:py35-lint]
-commands = bash .ci/flake8_wrapper.sh
+[testenv:py27-lint-docstring]
+commands = bash .ci/flake8_wrapper_docstrings.sh --exclude
 whitelist_externals = bash
+skip_install = True
```

Comment:
@jmchilton Should we add `skip_install = True` to all lint envs?

Reply:
Yes I think so - I'll follow this PR up with one that does that and takes care of the dev dependencies - they shouldn't be defined in this file - they should go in the global dev dependencies for Galaxy.